package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "ts-node example/index.ts",
     "build": "yarn build:compile && yarn build:swagger && yarn build:client && yarn build:license",
-    "build:compile": "tsup",
+    "build:compile": "tsup && mv dist/index.d.mts dist/esm/index.d.ts",
```

Comment:
the file structure is inconsistent.
couldn't use `onSuccess` for fixing that.
the corresponding issue: https://github.com/egoist/tsup/issues/938